crates/sncast/src/starknet_commands/account/delete.rs

```diff
@@ -48,12 +53,14 @@ pub fn delete(
         serde_json::from_str(&contents).expect("failed to read file { path }");
```

Comment:
Not added in this PR, but does `{ path }` work correctly here?

Reply:
I think so, bo the message is terrible tbh; it says very little to nothing.